### PR TITLE
SEO-181194-Xamarin.Forms-Map-Markers-Coverage-Issues

### PR DESCRIPTION
**Pull request by Faith-Atieno**
Hi @gowthamanthavasiyappan,
Pleasse review the changes made
Regards,
Faith

### CODE REVIEW

File: Xamarin/Maps/Markers.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ The shape of a marker icons can be customized using the [`MarkerIcon`](https://h
 
 {% endtabs %}
 
-![Marker Icon Image](Images/Marker_icon_type.png)
+![Marker icon image in Xamarin.Forms Maps.](\workspace\Content\xamarin-docs\Spell-Checker\Xamarin/Maps/Markers.md)
```
Comment (gowthamanthavasiyappan) [inline Xamarin/Maps/Markers.md:157]: Revert this type of change for all places and update proper path.

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

**Comment by Faith-Atieno** [comment]
rebuild

File: Xamarin/Maps/Markers.md (hunk 1 above)
Comment (Faith-Atieno) [reply Xamarin/Maps/Markers.md]: Hi @gowthamanthavasiyappan, kindly am done

**Pull request closed without merging**